src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx

```diff
@@ -9,56 +13,60 @@ interface Props {
   isSvg?: boolean;
 }
 
-const LegendItemChart: React.FC<Props> = ({ className, isSvg = true, title, color }) => (
-  <Container className={className}>
-    <ContainerIcon isSvg={isSvg} color={color}>
-      {isSvg ? <CircleLegendChart fill={color} /> : <Circle color={color} />}
-    </ContainerIcon>
-    <Title>{title}</Title>
-  </Container>
-);
+const LegendItemChart: React.FC<Props> = ({ className, isSvg = true, title, color }) => {
+  const { isLight } = useThemeContext();
+  return (
+    <Container className={className} isLight={isLight}>
+      <ContainerIcon isSvg={isSvg} color={color}>
+        {isSvg ? <CircleLegendChart fill={color} /> : <Circle color={color} />}
+      </ContainerIcon>
+
+      <Title isLight={isLight}>{title}</Title>
+    </Container>
+  );
+};
 
 export default LegendItemChart;
 
-const Container = styled('div')(({ theme }) => ({
+const Container = styled.div<WithIsLight>({
```

Comment:
`WithIsLight` is not used here so it should not be necessary

Reply:
done